ste/sender-blockBlobFromURL.go

```diff
@@ -69,9 +69,8 @@ func (c *urlToBlockBlobCopier) GenerateCopyFunc(id common.ChunkID, blockIndex in
 		setPutListNeed(&c.atomicPutListIndicator, putListNotNeeded)
 		return c.generateCreateEmptyBlob(id)
 	}
-	// Small blobs from all sources will be copied over to destination using PutBlobFromUrl with the exception of files
```

Comment:
Should we add a test that then checks the block list after validation? should be empty I guess?